File: java/week1/2.java

public class Sum{

	int sum(int ...a){
	int s = 0;
	for ( int i : a ){
	s = s + i;
	}
	return s;

	}
}
class in{


public static void main(String[] args) {
		Sum obj = new Sum();
		int a[] = {1,2,3,4,5};
		System.out.println(obj.sum(a));
	}
}

File: cpp/week1/2.cpp
#include <bits/stdc++.h>
using namespace std;

class HumanBeing
{
 public : void display()
 {
 	cout<<"Inside Member function\n";
 }
};

int main()
{
	//makes code faster...
	ios_base::sync_with_stdio(false);
	cin.tie(0);
	cout.tie(0);
	
	HumanBeing obj;
	obj.display();
} 
